### DISCUSSION TITLE
Add "Begin certificate"

### DISCUSSION THREAD

**Discussion by TheYigidOS** [khoih-prog/AsyncMQTT_Generic#18] [Q&A]
hi how can we add certificate to this link like "Begin Certificate"
https://github.com/khoih-prog/AsyncMQTT_Generic/blob/main/examples/WT32_ETH01/FullyFeaturedSSL_WT32_ETH01/FullyFeaturedSSL_WT32_ETH01.ino

**Comment by TheYigidOS** [comment]
İs there any solution?

**Reply by khoih-prog** [reply]
It's simple and basic knowledge. You have to learn doing by yourself. You can also post for help on Arduino Forum.

**Reply by TheYigidOS** [reply]
sir i cant find any working solutions

**Comment by khoih-prog** [comment]
At least try ESP32-core [**WiFiClientSecure**](https://github.com/espressif/arduino-esp32/tree/master/libraries/WiFiClientSecure/examples/WiFiClientSecure) example to make SSL working on WT32_ETH01, then you'll know what are the steps to do.

If not, read and try more, e.g [**TrustAnchors**](https://github.com/khoih-prog/EthernetWebServer_SSL/blob/main/TrustAnchors.md) and [**EthernetWebServer_SSL**](https://github.com/khoih-prog/EthernetWebServer_SSL) library

**Comment by TheYigidOS** [comment]
thanks u
